### PR TITLE
Use Messerli.ComponentRoot nuget package

### PR DESCRIPTION
**Pull request by sutser**

### CODE REVIEW

**Comment by sutser** [comment]
> 
> 
> Add changes to changelog addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/27686502e187d4873b2a6d325b9d8b2c111d5f36

File: Test.Utility/MockModuleBuilder.cs [hunk 1]
```diff
@@ -1,51 +1,16 @@
...
-using System.Collections.Generic;
-using Autofac;
-using Autofac.Core;
+﻿using Messerli.CompositionRoot;
 using Moq;
 
 namespace Messerli.Test.Utility
 {
-    public class MockModuleBuilder
+    public static class MockModuleBuilder
```
Comment (dngroth) [inline Test.Utility/MockModuleBuilder.cs]: Rename class to something with ´Extension´ in the name i. E ´ModuleBuilderMockExtension´

File: Test.Utility/MockContainerBuilder.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+﻿using Autofac;
+using Moq;
+
+namespace Messerli.Test.Utility
+{
+    public static class MockContainerBuilder
```
Comment (dngroth) [inline Test.Utility/MockContainerBuilder.cs]: Rename class to something with ´Extension´ in the name i. E ´ContainerBuilderMockExtension´

File: changelog.md [hunk 3]
```diff
@@ -26,3 +26,7 @@
 
 ## 0.5.2
 - Fix `RegisterModule(Mock<T> mock)` by removing `new()` restriction
+
+## Unreleased
+- Added `MockContainerBuilder`.
```
Comment (dngroth) [inline changelog.md]:
```suggestion
- Added `ContainerBuilderMockExtension`.
```

File: Test.Utility/MockModuleBuilder.cs (hunk 1 above)
Comment (sutser) [reply Test.Utility/MockModuleBuilder.cs]: Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/4d33a0c474230aebf9a54c1d4d870ecb7c356783

File: Test.Utility/MockContainerBuilder.cs (hunk 2 above)
Comment (sutser) [reply Test.Utility/MockContainerBuilder.cs]: Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/4d33a0c474230aebf9a54c1d4d870ecb7c356783

File: changelog.md (hunk 3 above)
Comment (sutser) [reply changelog.md]: Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/4d33a0c474230aebf9a54c1d4d870ecb7c356783

File: changelog.md [hunk 4]
```diff
@@ -26,3 +26,8 @@
 
 ## 0.5.2
 - Fix `RegisterModule(Mock<T> mock)` by removing `new()` restriction
+
+## Unreleased
```
Comment (dngroth) [inline changelog.md]: - Using `Messerli.CompositionRoot`
- Converted `MockModuleBuilder`s mock functionality to `ModuleBuilderMockExtension`
- Renamed `MockContainerBuilder` to `ContainerBuilderMockExtension`.
- Delete `MockModule` and `MockModuleBuilder`

Comment (sutser) [reply changelog.md]: Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/017bbea67da2efd3d482fb4328d281249606b7f2
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/dd4e600708c40895dc3311ea4071e4861f104b75

**Pull request merged**